### PR TITLE
[skip ci] docs: add instructions for installing Ansible

### PR DESCRIPTION
**Pull request by ktdreyer**
Add RPM and DEB instructions for installing Ansible.

### CODE REVIEW

**Comment by vasukulkarni** [comment]
thanks @ktdreyer

File: docs/source/index.rst [hunk 1]
```diff
@@ -25,6 +25,34 @@ You can install directly from the source on github by following these steps:
...
+Ansible on RHEL and CentOS
+--------------------------
+You can acquire Ansible on RHEL and CentOS by installing from
+`"Extras"<https://access.redhat.com/solutions/912213>`_.
+
+On RHEL::
+
+    subscription-manager repos --enable=rhel-7-server-extras-rpms
+
+(CentOS does not use subscription-manager and already has "Extras" enabled by default.)
```
Comment (vasukulkarni) [inline docs/source/index.rst]: I saw your other PR that talks abut ansible version 2.3.1, I believe the latest centos 7.3 and 7.4 would have that as default.

**Comment by leseb** [comment]
@ktdreyer this PR is perfect example of PR that should have `[skip ci] ` in their title :). For next time ;).
Thanks!

**Comment by ktdreyer** [comment]
I was just wondering why the CI was red ;)

**Pull request merged**